todo-angular/angular-featurehub-app/src/app/app.module.ts

```diff
@@ -1,8 +1,21 @@
-import { NgModule } from '@angular/core';
+import { APP_INITIALIZER, NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { FeatureToken, FEATURE_FLAG_TOKEN } from './feature-token';
+import { FeatureHubFlagService } from './featuerhub-service';
```

Comment:
could you please fix typo './featuerhub-service'  to './featurehub-service' and rename that file accordingly. Thanks!